docs/en/ingest-management/elastic-agent/configuration/env/shared-env.asciidoc

```diff
@@ -54,8 +54,8 @@ Overrides `FLEET_HOST` when set.
 [id="env-{type}-kibana-fleet-username"]
 `KIBANA_FLEET_USERNAME`
 
-| (string) The basic authentication username used to connect to {kib} and enable {fleet}.
-Overrides `KIBANA_USERNAME` when set.
+| (string) The basic authentication username used to connect to {kib} and retrieve a `service_token` to enable {fleet}.
```

Comment:
@dedemorton, I've changed the behaviour of `KIBANA_FLEET_USERNAME` and `ELASTICSEARCH_USERNAME` with https://github.com/elastic/beats/pull/29651, should I update the deprecation tags I added for 7.17 to reflect this?

Reply:
Yes, if you don't mind. Thanks!